packages/i18n/README.md

```diff
@@ -20,21 +20,27 @@ $ npm install --save @commercetools-frontend/i18n
 - `fr-FR`
 - `zh-CN`
 
-### Usage
+## Usage
 
-> This package should not be used directly to load the application messages, the `application-shell` does that internally.
+> The `<AsyncLocaleData>` component, or the `useAsyncLocaleData` hook, are internally used by the `<ApplicationShell>` and there is no need to use them directly.
+> If you do need to load translation files asynchronously, we expose an additional hook `useAsyncIntlMessages` that you can use for that.
+> The difference between those is that the "async locale data" components additionally load the application-kit and ui-kit messages, as well as the moment locales.
+
+### Using a React component with render props
+
+**With static messages**
 
 ```js
+import { IntlProvider } from 'react-intl';
```

Comment:
This change is unrelated?

Reply:
Yes and now. Since I'm updating the README though, to avoid confusion I think it would be better to keep the example generic and use `IntlProvider`, not our internal `ConfigureIntlProvider`.